pytests/conftest.py

```diff
@@ -101,36 +101,44 @@ def __init__(self, cli_args=None):
             self.config.update(cli_args)
         self.config['distribution'] = os.environ.get('DIST', 'photon')
         script = os.path.join(self.config['test_path'], 'repo/setup-repo.sh')
-        self.run([ 'sh', script, self.config['repo_path'] ])
+        self.run(['sh', script, self.config['repo_path']])
         self.tdnf_config = configparser.ConfigParser()
-        self.tdnf_config.read(os.path.join(self.config['repo_path'], 'tdnf.conf'))
+        self.tdnf_config.read(os.path.join(self.config['repo_path'],
+                                           'tdnf.conf'))
 
         # check execution environment and enable valgrind if suitable
         self.check_valgrind()
 
     def version_str_to_int(self, version):
         version_parts = version.split('.')
-        return version_parts[0] * 1000 + version_parts[1] * 100 + version_parts[2]
+        version = version_parts[0] * 1000
+        version += version_parts[1] * 100
+        version += version_parts[2]
+        return version
```

Comment:
Why are you doing this? I don't think it makes it any more readable.

Reply:
I will revert this change. At first I was trying to keep the line length less than 80 :)